packages/grafana-e2e/src/flows/addDashboard.ts

```diff
@@ -31,6 +31,7 @@ interface AddVariableOptional {
   label?: string;
   query?: string;
   regex?: string;
+  variableQueryForm?: (config: AddVariableConfig) => void;
```

Comment:
these changes are very rad. I wonder if it would be possible to document some of these amazing features better so that they are more discoverable.

Like in an ideal world I would love to be able to start typing addDashboard and then it would talk me through what I can pass through it and that if I pass in variableQueryForm I can test template variables or something you know? I feel like this should be possible via comments if they are properly formatted, I think we have something with jsdom style comments maybe? Or maybe I'm making that up.

Reply:
I agree with you here, a JSDOM style doc would be really nice. I'll see what's possible 😊